tests/components/srp_energy/test_config_flow.py

```diff
@@ -23,8 +25,9 @@
 from tests.common import MockConfigEntry
 
 
+@pytest.mark.usefixtures("mock_srp_energy_config_flow")
 async def test_show_form(
-    hass: HomeAssistant, mock_srp_energy_config_flow: MagicMock, capsys
+    hass: HomeAssistant, capsys: pytest.CaptureFixture[str]
```

Comment:
Why was the `mock_srp_energy_config_flow` removed from the signature? I think this fixture setups default values in the test.

Reply:
Not removed... just moved to `pytest.mark.usefixtures` above.